src/state/client/README.md

```diff
@@ -0,0 +1,18 @@
+## Client state
+
+---
+
+**TBD**: Client state is currently managed through `React.useState` and prop drilling. This README outlines how we will transition to Zustand in a future update.
+
+---
+
+Client state represents state data that is changeable by a user via UI components but is ephemeral
+and not stored on the server. We manage this state via the Zustand library.
+
+There are two stores, one holds information specific to station filtering for the map, provided to
+react components via a `useStationsStore()` hook. Because of the complexity of this area and the
+potential needs for using transitions it was speperated out.
+
+The other store (`useStore()`) holds other global state information. This is currently small as
+server state objects have been removed off into react query hooks. See
+[query hooks](../query-hooks/README.md) for more details.
```

Comment:
Suggest removing this block of information as it is specific to the station data portal (I think?)

Note additionally that not all client state need go into Zustand. React.useState does its job well, it just doesn't share state well between components in a large tree.